# Parameter settings for Ghost arduino controller

arduino:

# Servo Parameters
  str_centre: 100
  str_max: 140
  str_min: 60
  thr_centre: 87
  thr_max: 135
  thr_min: 53
